contracts/test/test_rental.js

```diff
@@ -0,0 +1,89 @@
+const Rental = artifacts.require("../contracts/Rental.sol");
+const chai = require('chai')
+const chaiAsPromised = require('chai-as-promised');
+
+
```

Comment:
Why are there that many empty lines? Maybe reduce to single empty line.

Reply:
Fixed